src/briefcase/commands/create.py

```diff
@@ -292,16 +292,19 @@ def _unpack_support_package(self, support_file_path, support_path):
         except (shutil.ReadError, EOFError) as e:
             raise InvalidSupportPackage(support_file_path) from e
 
-    def install_app_support_package(self, app: BaseConfig):
+    def install_app_support_package(self, app: BaseConfig, update: bool = False):
         """Install the application support package.
 
         :param app: The config object for the app
+        :param upsate: The force update the support package
         """
         try:
             support_path = self.support_path(app)
         except KeyError:
             self.logger.info("No support package required.")
         else:
+            if update and support_path.exists() and support_path.is_dir():
+                support_path.rmdir()
```

Comment:
`Path.rmdir()` requires that the directory is empty. You can use `shutil.rmtree(path)` instead; that will purge the directory before removing it. 

It's also a good idea to add some logging output so it's clear (to the user, and to the logs) that an action has been performed.